docs/source/dataframe-create.rst

```diff
@@ -1,8 +1,11 @@
-Create and Store Dask DataFrames
-================================
+Dask DataFrames: How to Create a Dataframe and Store It
+=======================================================
 
-Dask can create DataFrames from various data storage formats like CSV, HDF,
-Apache Parquet, and others.  For most formats, this data can live on various
+.. meta::
+    :description: Learn how to create dataframes and store them. Create a Dask DataFrame from various data storage formats like CSV, HDF, Apache Parquet, and others.
```

Comment:
```suggestion
    :description: Learn how to create and store DataFrames. Create a Dask DataFrame from various data storage formats like CSV, HDF, Apache Parquet, and others.
```

Reply:
Thanks @jrbourbeau! Both the meta tag and title are SEO-related updates. I also prefer the original title, but apparently `Create DataFrame` and similar search terms (e.g. `create a dataframe`, `create an empty dataframe`) are much more popular than `Store Dask DataFrames`, so the suggested title is rearranged based around putting `create dataframe` in the title (the earlier the better).

Same advice goes for the perhaps slightly awkward wording in the meta description.

Reply:
Ah, I see. Thanks for clarifying. What if we keep the original title but use the more popular search terms in the meta tag? Do you have a sense for how impactful that would be? The page title is pretty prominently displayed (both as the page header and in the TOC on the left sidebar) so I'm wondering if there's a happy medium where we can improve SEO for this page, but still avoid slightly awkward wording in the title

Reply:
Changing the title is the most impactful SEO change, but changing the first sentence of the page and the meta description will also help quite a bit, especially considering we're going from no SEO to some :). I think that's a fair compromise.

As an aside (and for improving SEO for additional dask docs pages) I wonder if we could use some [custom jinja template magic](https://stackoverflow.com/questions/45177750/sphinx-documentation-define-but-hide-heading-title-text) to override the title that search engines will look for?

Reply:
> Changing the title is the most impactful SEO change, but changing the first sentence of the page and the meta description will also help quite a bit, especially considering we're going from no SEO to some :). I think that's a fair compromise.

Okay, great. I'm looking forward to seeing what kind of impact changes like this will have

> I wonder if we could use some [custom jinja template magic](https://stackoverflow.com/questions/45177750/sphinx-documentation-define-but-hide-heading-title-text) to override the title that search engines will look for?

Neat idea! I think exploring options like this in the future will be really useful as we continue to improve our SEO (cc'ing @jacobtomlinson @jsignell for visibility)